lighthouse-core/audits/user-timings.js

```diff
@@ -118,6 +118,7 @@ class UserTimings extends Audit {
       category: 'Performance',
       name: 'user-timings',
       description: 'User Timing marks and measures',
+      helpText: 'Instrument your app with the User Timing API to measure its real-world performance. <a href="https://developers.google.com/web/tools/lighthouse/audits/user-timing" target="_blank" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
maybe we should emphasize this as a way to get custom, specific measures in? e.g. Eric knows that chromestatus's features appearing on screen is maybe the most important loading event for the site, but Lighthouse has no special knowledge of this.

By executing `performance.mark()` when they're loaded, he can get the timing for the thing he really cares about and wants to track in his Lighthouse report